api/filter/http/gzip.proto

```diff
@@ -74,4 +74,12 @@ message Gzip {
   // Allows disabling compression if response contains "last-modified"
   // header. Default is false.
   google.protobuf.BoolValue disable_on_last_modified = 8;
+
+  // Value from 9 to 15 that represents the base two logarithm of the compressor's window size.
+  // Larger values result in better compression at the expense of memory usage; e.g. 12 will produce
+  // a 4096 bytes window. Default is 15.
+  // Note that due to a known bug in the underlying zlib library, window bits with value 8 does not
+  // work as expected, therefore 9 is the smallest window size supported by this filter at the
+  // moment. This issue might be solved in future releases of the library.
```

Comment:
Whoops, my specific comment on this was lost. But just add a link here to the relevant section of the zlib manual if you can. Thanks.

Reply:
👍